### ISSUE TITLE
LDAP Support

### ISSUE THREAD

**Issue by HitmanH** [sapplica/sentrifugo#9]
Wondering what the time-scale to supporting this is?  One of the last things needed before we're ready to adpot
(and unfortunately not a developer hence can't do myself)

**Comment by admered1** [comment]
This query is closing rapidly on its second birthday. Is there any intent to support LDAP at some point ?

**Comment by kelderek** [comment]
Agreed - LDAP is likely to be quite important to businesses looking for something like this.  It makes it quite a bit more marketable for relatively little effort.
